### DISCUSSION TITLE
response.headers.status is no longer set

### DISCUSSION THREAD

**Discussion by jeffwilcox** [octokit/octokit.js#2006] [General] [answered]
Hey,
Something we hit in our production systems after the latest round of package-lock updates (either on our side or on `rest.js`/core) is that the headers returned no longer includes `status`.

I could be insane. But posting this here in case others were expecting, or not expecting, this sort of thing...

**Comment by imwiss** [comment, marked as answer]
Hey @jeffwilcox 👋   This is a change from the GitHub side where the `status` header is no longer being returned as of this morning. The `status` header was previously included by the rack server we use and the newer version has removed it. The infrastructure team confirmed that we won't be able to roll this back and restore the header unfortunately.

The workaround is to use the HTTP status itself, rather than the `status` header. Apologies if this caused downtime for you, this was an unfortunate side effect on the upgrade and not how we would usually announce breaking changes.

cc @gr2m in case there are updates required in Octokit.

**Reply by jeffwilcox** [reply]
Good to know, appreciate the extra context. I never honestly inspected what was really happening too much to know whether it was artificially being brought in by the library or not.

**Reply by jeffwilcox** [reply]
And for @gr2m I think the core libraries are fine.

We have our own custom retry and etag-caching tech in place (our 304 handling is where we found our reliance on the status being in the headers).

**Comment by gr2m** [comment]
Use `response.status` instead of `response.headers.status`

https://runkit.com/gr2m/octokit-rest-js-2006
